### ISSUE TITLE
Add Quarkus branch

### ISSUE THREAD

**Issue by cgruver** [redhat-cop/spring-rest#58]
I have created a port of this project from Spring Boot to Quarkus.

I would like to push the code to a quarks branch in this project to allow for easy comparison of Spring Boot and Quarkus.

This does mean maintaining two parallel branches going forward, so this may be an anti-pattern.

Perhaps we should consider creating a quarkus-rest project instead.

The quarkus code is here:  https://github.com/cgruver/spring-rest/tree/quarkus

**Comment by cgruver** [comment]
@etsauer @sabre1041 ^^

How would you guys like to approach this?

**Comment by sabre1041** [comment]
@cgruver Personally, since this no longer spring focused, it would most likely be suited in a separate repository. This would allow us to avoid the issues with multiple branches

@etsauer thoughts?

**Comment by garethahealy** [comment]
@cgruver ; is something like this not already covered in the quarkus examples?

**Comment by cgruver** [comment]
@garethahealy You are correct, it is covered + a lot more.

What I was intending is to provide something to a Spring Boot developer that shows the comparison between a Spring Boot application and a Quarkus application.  That is one of our barriers to adoption.

I did something similar with a team previously.  I wrote a fairly complex app with REST endpoints, REST Client dependencies, and JPA persistence.  I wrote the app in both Quarkus and Spring Boot.  Unfortunately, that app contained some proprietary logic, so I can share it.

I am currently working on a similar app which is unencumbered so that I can share it in my Github page.  I've used it for a couple of lunch-n-learn sessions with teams who currently code with Spring Boot and want to know about Quarkus.
